HTML/mockup-minimo/index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Inconsolata|Playfair+Display|Ubuntu:300,400&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+    <title>Mockup 3</title>
+</head>
+
+<body>
+    <header>
+        <nav class="nav">
+            <div class="logo">minim&Oslash;</div>
+            <ul class="menu">
+                <li class="menu__item">
+                    lifestyle
+                </li>
+                <li class="menu__item">
+                    photodiary
+                </li>
+                <li class="menu__item">
+                    music
+                </li>
+                <li class="menu__item">
+                    travel
+                </li>
+            </ul>
+            <span class="header__menu-icon">&#9776;</span>
+        </nav>
+        <picture class="header__image">
+            <source media="(min-width: 1200px)" srcset="img/image_0@3x-min.png">
+            <source media="(min-width: 500px)" srcset="img/image_0@2x-min.png">
+            <img src="img/image_0-min.png" class="header__image" alt="article photo">
+        </picture>
+    </header>
+    <main class="container-articles">
+        <article class="article-main">
```

Comment:
if take a look at next class names you should have here `article` and modifier `article--main`